.gitlab/functional_test/system_probe.yml

```diff
@@ -18,36 +17,83 @@
     - when: manual
       allow_failure: true
   stage: functional_test
-  needs: ["tests_ebpf"]
   variables:
     AGENT_MAJOR_VERSION: 7
     DD_PIPELINE_ID: $CI_PIPELINE_ID-a7
   script:
     - bash -l tasks/run-test-kitchen.sh system-probe-test $AGENT_MAJOR_VERSION
 
-kitchen_centos_sysprobe:
-  extends: .kitchen_test_system_probe
+.kitchen_test_system_probe_x64:
+  extends:
+    - .kitchen_test_system_probe
+    - .kitchen_azure_location_north_central_us
+  needs: [ "tests_ebpf_x64" ]
+  variables:
+    KITCHEN_ARCH: x86_64
+
+.kitchen_test_system_probe_arm64:
+  extends:
+    - .kitchen_test_system_probe
+  needs: [ "tests_ebpf_arm64" ]
+  variables:
+    KITCHEN_ARCH: arm64
+    KITCHEN_PROVIDER: ec2
+    KITCHEN_EC2_INSTANCE_TYPE: "t4g.large"
+    KITCHEN_EC2_SUBNET: subnet-c18341ed
+    KITCHEN_EC2_SG_IDS: sg-0f5617ceb3e5a6c39
+    KITCHEN_EC2_IAM_PROFILE_NAME: ci-datadog-agent-e2e-runner
+    KITCHEN_EC2_SPOT_PRICE: on-demand
```

Comment:
Should we hardcode some of these in `test/kitchen/drivers/ec2-driver.yml`? Specially things like `KITCHEN_EC2_SUBNET`, `KITCHEN_EC2_SG_IDS`, `KITCHEN_EC2_IAM_PROFILE_NAME` or `KITCHEN_EC2_SPOT_PRICE` which seem unlikely to change from one test to another. 

The stuff that we don't want to hardcode should go as a mixin `.kitchen_ec2` in `.gitlab/kitchen_common/testing.yml` the same way we do with `.kitchen_azure_*` so we can reuse it in other tests as well.

Reply:
I know @derekwbrown was using the ec2 driver to launch kitchen tests from his dev machine, so I think moving these into `test/kitchen/drivers/ec2-driver.yml` would break that setup. We could definitely create a mixin.

I was also trying to add SUSE tests, which notably cannot use spot instances, which is why `KITCHEN_EC2_SPOT_PRICE` was separate.

Reply:
Yup, Derek wrote the `kitchen.genconfig` task initially, but now that we have added EC2 support to the CI I think it makes more sense that Derek's setup is the one that's set manually and the one that works on Gitlab the default, and not the other way around.

Reply:
Well, except that Derek's setup is not Derek's setup but something that more people might want to use potentially, and removing the values has the caveat that we won't have them anywhere. I've changed my mind, ignore my comment 😂